tests/setup-runtime-dependencies.sh

```diff
@@ -109,10 +109,17 @@ if [ ! -e $libInstallDir ]; then
     mkdir -p $libInstallDir
 fi
 
+# Query runtime Id
+rid=`$dotnetCmd --version | grep 'Runtime Id:' | sed 's/^ *Runtime Id: *//g'`    
```

Comment:
Thanks! It looks like dotnet --version still works on coreclr repo. I am wondering when it will change to dotnet --info.